src/screens/Login/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from "react";
+import {
+  FormGroup,
+  FormControl,
+  ControlLabel
+} from "react-bootstrap";
+import { Auth } from 'aws-amplify';
+import LoaderButton from '../../components/LoaderButton';
+import { LoginState, LoginProps } from './types';
+import "./index.css";
+
+class Login extends Component<LoginProps, LoginState> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
We don't need to initialize state inside `constructor` method. It could be declared as class property such as:
```
class Login extends ... {
   state = {
   }
   ...
}
```